### DISCUSSION TITLE
How to add a specific folder to external option?

### DISCUSSION THREAD

**Discussion by chrber04** [egoist/tsup#787] [Q&A]
Hi, just recently started using tsup and I was curious how you add a specific folder to the external option?

We have a 'generated' folder and would like to exclude just this folder. It's at the same level in the directory as the tsup.config.ts file. 

Right now we exclude it with the RegExp /generated/

This caused issues with alias paths however, when we name our alias path @generated/*...

Thanks!

**Comment by PredokMiF** [comment]
I am having a problem with Prisma. I'm importing:
`import { PrismaClient } from '../generated/prisma'`
and in the config I specify
`external: ['../generated/prisma'],`

**Reply by SupertigerDev** [reply]
did you figure it out?

related: https://github.com/prisma/prisma/discussions/27008
